sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinBridgeEventHandlerImpl.java

```diff
@@ -120,8 +120,14 @@ public void handleRewind() {
   public void onScrub(ReadableMap percentage) {
     double percentValue = percentage.getDouble("percentage");
     percentValue = percentValue * 100.0f; // percentage * 100 so it can deal fine with milliseconds
-    float percent = (float) percentValue;
-    _player.seekToPercent(percent);
+    final float percent = (float) percentValue;
+    runOnUiThread(new Runnable() {
+      @Override
+      public void run() {
+        _player.seekToPercent(percent);
+      }
+    });
+
```

Comment:
Please check don't we need add same fix for handleRewind method also?

Reply:
@SergeyBicarte no, it is not necessary for handleRewind